src/index.html

```diff
@@ -3,12 +3,65 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="true"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <a href="#home" class="moyo">
+        <img
+          src="images/logo.png"
+          alt="Moyo"
+          class="moyo__img"
+        >
+      </a>
+
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a href="#home" class="is-active nav__link">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#SAMSUNG" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#SMARTPHONES" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#LAPTOPS_&_COMPUTERS"
+              class="nav__link"
+              data-qa="hover">Laptops & computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#GADGETS" class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#TABLETS" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#PHOTO" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="#VIDEO" class="nav__link">Video</a>
+          </li>
+        </ul>
+
+
```

Comment:
Consider removing the redundant line here, use just one line to have space between blocks